tests/behat/features/capabilities/groups/flexible/groups-flexible-page-access.feature

```diff
@@ -0,0 +1,48 @@
+@api @javascript
+Feature: Validate accessibility of flexible group pages for different users
+
+  Background:
+    Given I enable the module "social_group_flexible_group"
+    And I am logged in as an "sitemanager"
+    And users:
+      | name        | mail                    | status | roles       |
+      | Behat Owner | behat_owner@example.com | 1      | sitemanager |
+      | Non Member  | non_member@example.com  | 1      |             |
+      | Member      | member@example.com      | 1      |             |
+
+    And groups no validation:
+      | label           | type           | author      | field_flexible_group_visibility |
+      | Test Challenge  | flexible_group | Behat Owner | public                          |
```

Comment:
I would prefer to not use title Challenge for Flexible groups.

Reply:
@ribel fixed